tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -2179,5 +2179,75 @@ public void TestBinaryOperationsOnExplodedNumericColumns()
             Assert.True(reverseInPlace.ElementwiseEquals(ints).All());
             Assert.False(reverseInPlace.ElementwiseEquals(reverse).All());
         }
+
+        [Fact]
+        public void GetColumnTests()
+        {
+            DataFrame dataFrame = MakeDataFrameWithAllColumnTypes(10);
+            PrimitiveDataFrameColumn<int> primitiveInts = dataFrame.Columns.GetPrimitiveColumn<int>("Int");
```

Comment:
I wonder if we need to put `Column` in the method name.... We are already on a `Column` collection.

```
dataFrame.Columns.GetPrimitive<int>("Age");
dataFrame.Columns.GetInt32("Age");
dataFrame.Columns.GetSingle("Wage");

dataFrame.Columns.GetString("City");
```

Reply:
I'm not opposed to the idea. I wonder if we have naming guidelines here, and what the API review folks would accept

Reply:
Maybe let's leave it as-is for now, and make a proposal to drop `Column`, and see how folks feel about it.